src/disco/components/Addon/index.js

```diff
@@ -242,6 +244,30 @@ export class AddonBase extends React.Component {
     }
   };
 
+  installStaticTheme = async (event) => {
```

Comment:
Originally I had this outside of the class so I could spy on it being called but I decided to move back inside the class b/c I couldn't seem to spy on the install and enable methods which I thought was more import to spy on (rather than the function itself). hopefully this is okay..